### DISCUSSION TITLE
Exit code does not persist on an empty command in Nushell v0.69.1

### DISCUSSION THREAD

**Discussion by lewis-yeung** [JanDeDobbeleer/oh-my-posh#2872] [General]
In Nushell v0.69.1, if we press <kbd>Enter</kbd> on an empty command, a non-zero exit code will not persist in the next spawned prompt as expected, instead it will be always reset to 0. Can we do something in the init script to make the logic consistent with other shells?

**Comment by JanDeDobbeleer** [comment]
@lewis-yeung probably, let's have a look at that.

**Reply by JanDeDobbeleer** [reply]
Had a glance, this is a request for nushell, not us.

**Reply by lewis-yeung** [reply]
You're right.

**Comment by lewis-yeung** [comment]
I created an issue in Nushell repo: nushell/nushell#6673.

**Comment by lewis-yeung** [comment]
This has been resolved in Nushell v0.70.0.
